powersimdata/design/investment/investment_costs.py

```diff
@@ -50,39 +48,29 @@ def calculate_ac_inv_costs(
     scenario,
     sum_results=True,
     exclude_branches=None,
-    base_grid=None,
 ):
     """Calculate cost of upgrading AC lines and/or transformers in a scenario.
     NEEM regions are used to find regional multipliers.
 
     :param powersimdata.scenario.scenario.Scenario scenario: scenario instance.
     :param bool sum_results: whether to sum data frame for each branch type. Defaults to
         True.
-    :param powersimdata.input.grid.Grid base_grid: a Grid to compare against. If None,
-        the grid model and interconnect from the ``scenario`` are used to instantiate a
-        corresponding unmodified Grid.
     :return: (*dict*) -- keys are {'line_cost', 'transformer_cost'}, values are either
         float if ``sum_results``, or pandas Series indexed by branch ID.
         Whether summed or not, values are $USD, inflation-adjusted to today.
     """
     grid = scenario.state.get_grid()
-    if base_grid is None:
-        base_grid = Grid(
-            scenario.info["interconnect"].split("_"), source=scenario.info["grid_model"]
-        )
```

Comment:
The logic `if base_grid is None` can be replaced by `base_grid = scenario.get_base_grid()`

Reply:
done